README.md

```diff
@@ -168,6 +170,9 @@ editorial work.
     [LGPL-2.1 license](https://choosealicense.com/licenses/lgpl-2.1/)
     because it is technically simpler to understand and abide by.
 
+  If neither of these two licenses can be used, we encourage using
+  another open source license approved by the OSI.
```

Comment:
Can we encourage choosing a license in the intersection of the OSI and FSF lists or is there another case to account for? This would also avoid taking sides.

Reply:
It sounds reasonable to encourage a license in the intersection. This notably means we do not encourage using CC-0, which is not OSI approved. It probably also "dismisses" some obscure licenses that are OSI approved, which is fine by me.